pm/recipient.go

```diff
@@ -22,6 +22,7 @@ var errInsufficientSenderReserve = errors.New("insufficient sender reserve")
 var maxWinProb = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 256), big.NewInt(1))
 
 var paramsExpirationBlock = big.NewInt(5)
+var paramsExpiryBuffer, _ = new(big.Float).Mul(new(big.Float).SetInt(paramsExpirationBlock), big.NewFloat(0.9)).Int64()
```

Comment:
A problem I see with this method of calculating a buffer before the expiration is that it explicitly depends on `paramsExpirationBlock` which is determined by O. While this is a hardcoded variable, there is no guarantee that B & O will have the same view of the value of this variable since they could be running different versions of go-livepeer [1]. For example, it could be the case that after #1873 `paramsExpirationBlock = 10` on B, but an O is still using an older version of go-livepeer where `paramsExpirationBlock = 5`).

What if we just return `ErrTicketParamsExpired` if we are within 1-2 blocks of the expiration block?

[1] I realize that this is also a problem for the [auth token](https://github.com/livepeer/go-livepeer/blob/master/server/broadcast.go#L846) buffer calculation

Reply:
> [1] I realize that this is also a problem for the auth token buffer calculation

Want to address this in a separate issue ?

Reply:
> Want to address this in a separate issue ?

👍